src/Fields/Fields.jl

```diff
@@ -437,11 +437,11 @@ ClimaCore.Fields.truncate_printing_field_types() = true
 =#
 truncate_printing_field_types() = false
 
-function Base.show(io::IO, ::Type{T}) where {T <: Fields.Field}
+function Base.show(io::IO, f::Fields.Field)
```

Comment:
What about
```suggestion
function Base.show(io::IO, T::Type{Fields.Field{V, S}) where {V,S}
```

Reply:
This still results in invalidations 🤔

Reply:
Ah, it looks like we can use `Base.show(io::IOContext, ::Type{<:Fields.Field})`. Is that alright, @simonbyrne?

Reply:
Hmm, there's still invalidations with that, but it's down from 1229 to 149 (at least in the MWE)

Reply:
Ok, and based on the invalidations script, this invalidation site will drop to the bottom of the list.